src/main/java/org/jabref/preferences/GuiPreferences.java

```diff
@@ -38,6 +39,10 @@ public class GuiPreferences {
     private final BooleanProperty mergeShouldShowUnifiedDiff;
     private final BooleanProperty mergeHighlightWords;
     private final BooleanProperty mergeShowChangedFieldsOnly;
+    private final BooleanProperty mergeApplyToAllEntries;
+
+    private final ObjectProperty<DuplicateResolverDialog.DuplicateResolverResult> allEntriesDuplicateResolverDecision;
+
     private final DoubleProperty sidePaneWidth;
 
     public GuiPreferences(double positionX,
```

Comment:
Maybe better in a new MergeDialogPreferences? Could also take mergeShouldShowInifiedDiff and HighlightWords?